celiagg/ndarray_canvas.hxx

```diff
@@ -332,16 +331,14 @@ void ndarray_canvas<pixfmt_t>::_draw_text_internal(const char* text, Font& font,
 
 template<typename pixfmt_t>
 template<typename base_renderer_t>
-void ndarray_canvas<pixfmt_t>::_draw_text_raster(GlyphIterator& iterator,
+void ndarray_canvas<pixfmt_t>::_draw_text_raster(const char* text,
     Font& font, const agg::trans_affine& transform, Paint& fillPaint,
     const GraphicsState& gs, base_renderer_t& renderer)
 {
 #ifdef _ENABLE_TEXT_RENDERING
     typedef FontCache::FontCacheManager::gray8_adaptor_type font_rasterizer_t;
     typedef FontCache::FontCacheManager::gray8_scanline_type scanline_t;
-
-    const bool eof = (gs.drawing_mode() & GraphicsState::DrawEofFill) == GraphicsState::DrawEofFill;
-    m_rasterizer.filling_rule(eof ? agg::fill_even_odd : agg::fill_non_zero);
```

Comment:
Drive-by cleanup. `m_rasterizer` isn't used by this method at all. The font cache manager supplies a rasterizer.